src/main/java/org/embulk/output/sftp/provider/sftp/SftpFileProvider.java

```diff
@@ -70,6 +73,7 @@ protected FileSystem doCreateFileSystem(final FileName name, final FileSystemOpt
                     fileSystemOptions);
         }
         catch (final Exception e) {
+            logger.info("Creat sftp connection fail: {}", e.getMessage());
```

Comment:
1. Better to log it in `warn` or `error` level instead of `info`?
2. Typo, `Creat` -> `Create`. Better to have full sentence e.g. `Create SFTP connection was failed: {}`?